docs/reference/installation.md

```diff
@@ -196,3 +196,100 @@ If you plan to use GPU-enabled functionalities, then you are advised to install
 **HINT:** All tests probe for the `TEST_DEVICE` enviromental variable when running.
 If this enviromental variable is set during testing, it allows for easily running all tests on a different device (e.g., setting `TEST_DEVICE=cuda:0` runs all tests on the first GPU of the system).
 
+## Build the docker images yourself _(optional)_
```

Comment:
Please remove this section and appropriately update `customize.md` file instead by adding only the instructions for NVIDIA embedded devices.

Reply:
Added the section to the appropriate file.